### PR TITLE
Add files for projects

### PR DESCRIPTION
**Pull request by pohlinwei**

### CODE REVIEW

File: portfolio/src/main/webapp/projects/adventure.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <header>
+      <div id="logo"/>
+    </header>
+    <main>
+      <h1 class="project-name">Adventure</h1>
+      <picture>
+        <source media="(max-width: 399px), (orientation: portrait)" srcset="adventure/0_mobile.png" size="400px">  
+        <source media="(min-width: 400px)" srcset="adventure/0.png" size="400px">
+        <img class="screenshot" src="adventure/0_mobile.png"" alt="Screenshot of Adventure">
+      </picture>      <picture>
```
Comment (fanwgwg) [inline portfolio/src/main/webapp/projects/adventure.html]: I think there are some formatting issues here

File: portfolio/src/main/webapp/projects/projects.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "name": "Adventure",
+        "image": "",
+        "description": "A Tumblr theme that is good for text-heavy content.",
+        "tools": "HTML",
+        "date": "Winter 2017",
+        "page": "projects/adventure.html",
+        "screenshots": "adventure",
+        "github": "https://github.com/pohlinwei/Adventure"
+    }
+]
```
Comment (fanwgwg) [inline portfolio/src/main/webapp/projects/projects.json]: EOF issue?

File: portfolio/src/main/webapp/projects/vectors/github.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
...defs><path d="M0 330.33C0 474.47 91.6 596.22 218.8 639.38C234.8 642.23 240.8 632.46 240.8 623.91C240.8 616.17 240.4 590.52 240.4 563.24C160 578.3 139.2 543.29 132.8 524.96C129.2 515.6 113.6 486.69 100 478.95C88.8 472.84 72.8 457.78 99.6 457.37C124.8 456.97 142.8 480.99 148.8 490.76C177.6 540.03 223.6 526.18 242 517.63C244.8 496.46 253.2 482.21 262.4 474.07C191.2 465.92 116.8 437.83 116.8 313.23C116.8 277.81 129.2 248.49 149.6 225.69C146.4 217.55 135.2 184.16 152.8 139.37C152.8 139.37 179.6 130.82 240.8 172.76C266.4 165.43 293.6 161.77 320.8 161.77C348 161.77 375.2 165.43 400.8 172.76C462 130.41 488.8 139.37 488.8 139.37C506.4 184.16 495.2 217.55 492 225.69C512.4 248.49 524.8 277.4 524.8 313.23C524.8 438.24 450 465.92 378.8 474.07C390.4 484.25 400.4 503.79 400.4 534.33C400.4 577.89 400 612.91 400 623.91C400 632.46 406 642.64 422 639.38C548.4 596.22 640 474.07 640 330.33C640 150.37 496.8 4.6 320 4.6C143.2 4.6 0 150.37 0 330.33Z" id="cKCtfBKzu"></path></defs><g><g><g><use xlink:href="#cKCtfBKzu" opacity="1" fill="#000000" fill-opacity="1"></use><g><use xlink:href="#cKCtfBKzu" opacity="1" fill-opacity="0" stroke="#000000" stroke-width="1" stroke-opacity="0"></use></g></g></g></g></svg>
```
Comment (fanwgwg) [inline portfolio/src/main/webapp/projects/vectors/github.svg:3]: Same EOF issue with this svg file

File: portfolio/src/main/webapp/projects/acompianist.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <header>
+      <div id="logo"/>
+    </header>
+    <main>
+      <h1 class="project-name">AComPianist</h1>
+      <picture>
+        <source media="(max-width: 399px), (orientation: portrait)" srcset="acompianist/0_mobile.png" size="400px">  
+        <source media="(min-width: 400px)" srcset="acompianist/0.png" size="400px">
+        <img class="screenshot" src="acompianist/0_mobile.png"" alt="Screenshot of AComPianist">
+      </picture>      <picture>
```
Comment (fanwgwg) [inline portfolio/src/main/webapp/projects/acompianist.html]: Html formatting issue, please fix

**Review by fanwgwg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/projects/adventure.html (hunk 1 above)
Comment (pohlinwei) [reply portfolio/src/main/webapp/projects/adventure.html]: Thanks for pointing this out!

**Review by fanwgwg — APPROVED** [review]

**Pull request merged**